quantiles/include/quantiles_sketch.hpp

```diff
@@ -146,14 +146,13 @@ namespace quantiles_constants {
 }
 
 template <typename T,
-          typename Comparator = std::less<T>,
+          typename Comparator = std::less<T>, // strict weak ordering function (see C++ named requirements: Compare)
           typename Allocator = std::allocator<T>>
 class quantiles_sketch {
 public:
-  using C = Comparator;
-  using A = Allocator;
-  using AllocDouble = typename std::allocator_traits<Allocator>::template rebind_alloc<double>;
-  using vector_double = std::vector<double, AllocDouble>;
+  using value_type = T;
```

Comment:
i don't think this is actually used, but i guess it gives us a consistent API and makes it easy to access

Reply:
I tied to follow some practices of STL. they have many type aliases like this for users